web/docs/guides/auth/auth-facebook.mdx

```diff
@@ -0,0 +1,116 @@
+---
+id: auth-facebook
+title: "Facebook Login"
+description: Setting up Facebook Login
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+### Setting up Facebook Login
+
+## Description
+
+Adding a Facebook Login button to your application is easy if you follow these steps.
+
+The setup process consists of three parts:
+
+* Create and configure a Facebook Application on the [Facebook Devalopers Site](https://developers.facebook.com)
+* Add your Facebook keys to your [Supabase Project](https://supabase.io)
+* Add the login code to your [Supabase JS Client App](https://github.com/supabase/supabase-js)
+
+## Environment
```

Comment:
Remove this whole section. For a new person reading this they might think, "Whats a Facebook Developers Dashboard"? The guide is going to get them set up with the environment, so we don't need to tell them up front